File: ios/Classes/PaymentResponse.swift

import Foundation

struct PaymentResponse: Codable {
    let status: String?
    let message: String?
    let errorcode: String?
    let data: PaymentResponseData?
}

struct PaymentResponseData: Codable {
    let ecommerceId: String?
    let auth_token: String?
}

File: android/src/main/kotlin/com/evertec/athmovil/athmovil_checkout_flutter/models/PaymentResponse.kt
package com.evertec.athmovil.athmovil_checkout_flutter.models

import com.evertec.athmovil.athmovil_checkout_flutter.models.Ecommerce
import com.google.gson.annotations.SerializedName

data class PaymentResponse (
    
    @SerializedName("status")
    var status: String = "",

    @SerializedName("data")
    var data: Ecommerce? = null,

    @SerializedName("message")
    var message: String = "",
    
    @SerializedName("errorcode")
    var errorcode: String = ""

)
